plejd/DeviceRegistry.js

```diff
@@ -171,6 +214,11 @@ class DeviceRegistry {
     return `${deviceId}_${outputIndex}`;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  getUniqueInputId(deviceId, inputIndex) {
+    return `${deviceId}_${inputIndex}`;
```

Comment:
this is not unique if there is an output with the same index. 

maybe use `${deviceId}_I_${inputIndex}`?

In retrospect, maybe output index should be changed also to be symmetric?

Reply:
Fixed in coming updated